editoast/src/client/mod.rs

```diff
@@ -77,6 +77,11 @@ pub struct RunserverArgs {
     #[derivative(Default(value = r#""".into()"#))]
     #[clap(long, env = "ROOT_PATH", default_value_t = String::new())]
     pub root_path: String,
+    /// A comma separated list of URIs from which are fetched the public keys
+    /// If present, authentication is enable and editoast will require a valid Bearer token for every call.
+    /// If absent, no authentication is required.
+    #[clap(long, env = "OIDC_JWK_URIS")]
+    pub oidc_jwk_uris: Option<String>,
```

Comment:
Can use value delimiter: https://docs.rs/clap/latest/clap/struct.Arg.html#method.value_delimiter

```suggestion
    /// A comma separated list of URIs from which are fetched the public keys
    /// If present, authentication is enable and editoast will require a valid Bearer token for every call.
    /// If absent, no authentication is required.
    #[clap(long, env = "OIDC_JWK_URIS")]
    #[arg(value_delimiter(','))] 
    pub oidc_jwk_uris: Vec<String>,
```